examples/network-load-balancer/variables.tf

```diff
@@ -0,0 +1,41 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# These variables are expected to be passed in by the operator
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable "project" {
+  description = "The project ID to create the resources in."
+  type        = string
+}
+
+variable "region" {
+  description = "The region to create the resources in."
+  type        = string
+}
+
+variable "zone" {
+  description = "The availability zone to create the sample compute instances in. Must within the region specified in 'var.region'"
+  type        = string
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# OPTIONAL PARAMETERS
+# These variables have defaults, but may be overridden by the operator.
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable "name" {
+  description = "Name for the load balancer forwarding rule and prefix for supporting resources."
+  type        = string
+  default     = "ilb-example"
+}
+
+variable "custom_labels" {
+  description = "A map of custom labels to apply to the resources. The key is the label name and the value is the label value."
+  type        = map(string)
+
+  default = {
```

Comment:
What about removing this default and moving adding to the test config so that users of the example don't get our test labels attached to their resources?

Reply:
We actually don't need those in the examples at all because of the naming scheme we're using. Might as well remove completely - they don't really add any value to the examples.